md/uid-modal-tutorial.md

```diff
@@ -1,5 +1,9 @@
 # How to create a modal window using CSS
 
+::: warning
+**:fa-exclamation-triangle: Warning:** This tutorial is deprecated after 7.8 because of the introduction of the modal container. For more information, read the [modal container](widgets.md#modal-widget) page.
```

Comment:
```suggestion
**:fa-exclamation-triangle: Warning:** This tutorial is deprecated starting with Bonita 7.8 because of the introduction of the modal container as a provided container. For more information, read the [modal container](widgets.md#modal-widget) page.
```